cordova-lib/spec-cordova/fixtures/plugins/sample-cocoapod-plugin-overlapping-dependency/RELEASENOTES.md

```diff
@@ -0,0 +1,141 @@
+<!--
```

Comment:
Will follow your suggestion to clone [https://github.com/apache/cordova-lib/tree/master/cordova-lib/spec-cordova/fixtures/plugins/test](https://github.com/apache/cordova-lib/tree/master/cordova-lib/spec-cordova/fixtures/plugins/test) and add the podspec framework tag to plugin.xml to replace the current fixtures. Thanks!